examples/keras_recipes/tensorflow_numpy_models.py

```diff
@@ -0,0 +1,340 @@
+"""
+Title: Writing Keras Models With Tensorflow NumPy
+Author: lukewood
+Date created: 2021/08/28
+Last modified: 2021/08/28
+Description: Overview of how to use the Tensorflow NumPy API to write Keras models.
+"""
+"""
+# Introduction
+[NumPy](https://numpy.org/) is a hugely successful python linear algebra library.
+Many machine learning practitioners prototype projects NumPy, converting to Jax or
+PyTorch at later stages.
+
+Tensorflow recently launched [tf_numpy](https://www.tensorflow.org/guide/tf_numpy), a
+TensorFlow copy of a subset of the NumPy API.  Thanks to `tf_numpy`, porting layers or
+models from NumPy to Keras is now easy!
+
+The Tensorflow NumPy API has full integration with the Tensorflow ecosystem.  Features
+such as automatic differentiation, Tensorboard, Keras model callbacks, TPU distribution
+and model exporting are all supported.
+
+Let's run through a few examples.
+"""
+
+"""
+# Setup
+Tensorflow NumPy requires TensorFlow 2.5 or later.
+"""
+
+import tensorflow as tf
+
+print(tf.__version__)
+
+import tensorflow as tf
+import tensorflow.experimental.numpy as tnp
+import keras
+import numpy as np
+
+"""
+Additionally, in order to use Tensorflow NumPy you must modify some global settings using
+`tnp.experimental_enable_numpy_behavior`.
+"""
+
+tnp.experimental_enable_numpy_behavior()
+
+"""
+To test our models we will use the Boston housing prices regression dataset.
+"""
+
+(x_train, y_train), (x_test, y_test) = tf.keras.datasets.boston_housing.load_data(
+    path="boston_housing.npz", test_split=0.2, seed=113
+)
+
+
+def evaluate_model(model: keras.Model):
+    [loss, meanAbsolutePercentError] = model.evaluate(x_test, y_test, verbose=0)
```

Comment:
Always use snake case for variable names, e.g. `percent_error`

Reply:
Done!

Reply:
Done